packages/core/lib/Bus.ts

```diff
@@ -122,6 +124,42 @@ export class Bus<A extends Actor<I, T, O>, I extends IAction, T extends IActorTe
     }
   }
 
+  /**
+   * Indicate that the given actor has the given actor dependencies.
+   *
+   * This will ensure that the given actor will be present in the bus *after* the given dependencies.
+   *
+   * @param {A} dependent A dependent actor.
+   * @param {A[]} dependencies Actor dependencies.
+   */
+  public addDependencies(dependent: A, dependencies: A[]) {
+    for (const dependency of dependencies) {
+      this.dependencyLinks.push({ dependent, dependency });
+    }
+    this.reorderForDependencies();
+  }
+
+  /**
+   * Reorder the bus based on all present dependencies.
+   */
+  public reorderForDependencies() {
```

Comment:
This can give incorrect results if there are chained dependencies, depending on the order.
Example: 
  dependencyLinks: [ B depends on C, A depends on B ]
  this.actors = [ A, B, C ]
  After running this function the result will be [ B, A, C ], while C should be before B.

Reply:
Yeah, I was a bit lazy in my initial implementation.
I've rewritten the whole thing with proper chained dependency support, and added a test for your case.